bbot/core/helpers/dns.py

```diff
@@ -44,7 +88,7 @@ def __init__(self, parent_helper):
             raise DNSError(f"Failed to create BBOT DNS resolver: {e}")
         self.timeout = self.parent_helper.config.get("dns_timeout", 5)
         self.retries = self.parent_helper.config.get("dns_retries", 1)
-        self.abort_threshold = self.parent_helper.config.get("dns_abort_threshold", 5)
+        self.abort_threshold = self.parent_helper.config.get("dns_abort_threshold", 50)
```

Comment:
@TheTechromancer Is this going to have a significant impact the wildcard detection?

Reply:
No it will not impact wildcards. Abort threshold has to do with aborting queries to a certain domain+rdtype after so many failed DNS queries. Its purpose is to prevent the scan from getting stuck on a black-holed domain or misconfigured DNS server.